scrapy/utils/python.py

```diff
@@ -174,17 +174,24 @@ def new_method(self, *args, **kwargs):
         return cache[self]
     return new_method
 
+
 _BINARYCHARS = {six.b(chr(i)) for i in range(32)} - {b"\0", b"\t", b"\n", b"\r"}
 _BINARYCHARS |= {ord(ch) for ch in _BINARYCHARS}
 
-
+@deprecated("scrapy.utils.python.binary_is_text")
 def isbinarytext(text):
-    """Return True if the given text is considered binary, or False
-    otherwise, by looking for binary bytes at their chars
+    """ This function has been removed.
+    Please use scrapy.utils.python.binary_is_text """
```

Comment:
I had considered it. But I wasn't sure if the old function doc string would get checked by anyone (unless checking the source and then it should be obvious), and adding a note to the new function doc about an unrelated function would seem strange as well.

I'll add a note to the old one.